src/plugins/actionmng/actionmng.js

```diff
@@ -1,9 +1,3 @@
-/**
- * @title WET-BOEW Action Manager
- * @overview API that coordinate actions with other wet-boew plugin
- * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
- * @author @duboisp
- */
```

Comment:
Re-add the previous comment block, feel free to add you name in the "author" section

Reply:
completed as requested